cogeo_mosaic/backends/base.py

```diff
@@ -17,7 +17,7 @@ class BaseBackend(AbstractContextManager):
     path: str
     mosaic_def: MosaicJSON
     _backend_name: str
-    _file_byte_size: Optional[int]
+    _file_byte_size: Optional[int] = 0
```

Comment:
We have to set a default value here because when the backend hit the cache we cannot set the size.